dask_cuda/tests/test_explicit_comms.py

```diff
@@ -18,6 +19,13 @@
 from dask_cuda.initialize import initialize
 from dask_cuda.utils import get_ucx_config
 
+if sys.version_info.minor < 8:
+    pytest.skip(
+        "Temporarily skipping some tests because of a bug "
+        "in Dask see <https://github.com/rapidsai/dask-cuda/issues/746>"
+    )
```

Comment:
```suggestion
pytest.mark.skipif(sys.version_info.minor < 8,
    reason="Temporarily skipping some tests because of a bug "
    "in Dask see <https://github.com/rapidsai/dask-cuda/issues/746>"
)
```